docs/v2-system-addon/vendor.md

```diff
@@ -0,0 +1,44 @@
+# Vendoring in npm libraries
+
+#### Build system basics
+
+#### Vendored libraries as externals
+
+#### Is this library a good candidate to depend on and vendor?
+
+#### How to vendor in a library
+
+- [ ] file a bug for vendoring in the library itself
+
+- [ ] Get license approval from the licensing team
+
+  - [ ] review the mozilla-central policy at
```

Comment:
Are these steps for new licenses that weren't previously used or should we follow this step with any new package? 

A note on BSD 3-Clause since its not explicitly mentioned on the license page but rather that "MIT, New BSD, and similar permissive licenses" are allowed.

Another source of truth we can look (suggested by @mhoye) at is https://mana.mozilla.org/wiki/pages/viewpage.action?pageId=87365351 or https://docs.google.com/document/d/1Oguqp43W4_ChyroJ9AJAzG1jSwkUWfKvBKVvrDxVsMg/edit.

There's a specific bullet there mentioning 
```
The following licences are compatible with the MPL, and code under these licenses can be integrated into Mozilla products
* ...
* The New BSD and 2- and 3-clause BSD licenses
```

And I'll still need to do the remaining steps to create a patch for license.html to add this new package usage there.

Reply:
The licensing page says

> Remember, always consult the licensing team before importing Third Party Code.

so I think it's always necessary to get their approval, and the other information is just there to tell you what to expect and which licenses are known to be compatible.

According to OSI, "New BSD" == "BSD 3 Clause"; I'll add some clarifying text related to that too.

I believe these steps need to be followed for all libraries.

Reply:
Ah makes sense to always consult and follow these steps